src/NzbDrone.Core.Test/OrganizerTests/FileNameBuilderTests/FileNameBuilderFixture.cs

```diff
@@ -752,13 +750,13 @@ public void should_not_update_media_info_if_token_configured_and_revision_is_new
         }
 
         private void GivenMediaInfoModel(string videoCodec = "h264",
-            string audioCodec = "dts",
-            int audioChannels = 6,
-            int videoBitDepth = 8,
-            HdrFormat hdrFormat = HdrFormat.None,
-            string audioLanguages = "eng",
-            string subtitles = "eng/spa/ita",
-            int schemaRevision = 5)
+                                         string audioCodec = "dts",
+                                         int audioChannels = 6,
+                                         int videoBitDepth = 8,
+                                         HdrFormat hdrFormat = HdrFormat.None,
+                                         string audioLanguages = "eng",
+                                         string subtitles = "eng/spa/ita",
+                                         int schemaRevision = 5)
```

Comment:
I am not sure about this part; the rest looks fine though - thoughts from others?